repository/repo-operations.go

```diff
@@ -405,3 +453,20 @@ func openPullRequest(config *config.GitXargsConfig, repo *github.Repository, bra
 	config.Stats.TrackPullRequest(repo.GetName(), pr.GetHTMLURL())
 	return nil
 }
+
+// Returns true if a pull request already exists in the given repo for the given branch
+func pullRequestAlreadyExistsForBranch(config *config.GitXargsConfig, repo *github.Repository, branch string, repoDefaultBranch string) (bool, error) {
+	opts := &github.PullRequestListOptions{
+		// Filter pulls by head user or head organization and branch name in the format of user:ref-name or organization:ref-name
+		// https://docs.github.com/en/rest/reference/pulls#list-pull-requests
+		Head: fmt.Sprintf("%s:%s", *repo.Owner.Login, branch),
```

Comment:
NIT: could use the `*repo.GetOwner().Login` method here for consistency.

Reply:
Done: https://github.com/gruntwork-io/git-xargs/pull/34/commits/2a875e8d0f934aa76f9552c3623f8d6084d0a315